manifests/init.pp

```diff
@@ -313,6 +316,12 @@
   include foreman::database
   contain foreman::service
 
+  if $facts['os']['selinux']['enabled'] {
+    File {
+      selinux_ignore_defaults => $foreman::selinux_ignore_defaults,
+    }
+  }
```

Comment:
I'm against this. It's way too broad and magic. Pretty sure this affects the whole catalog, not just the files created by this module.

Instead, can we set the property everywhere? It's more verbose, but also easier to follow.